fia_api/routers/jobs.py

```diff
@@ -363,7 +360,7 @@ async def create_autoreduction(
     """
     user = get_user_from_token(credentials.credentials)
     if user.user_number != -1:  # API Key user has psuedo user number of -1
-        raise HTTPException(status_code=HTTPStatus.FORBIDDEN)
+        raise AuthError()
```

Comment:
Can you add message like `f"User number: {user.user_number} attempted to create autoreduction - autoreduction only creatable via APIKey"`